server.js

```diff
@@ -1,48 +1,58 @@
+/**
+ * Module dependencies
+ */
 const BrowserWindow = require('browser-window')
 const assert = require('assert')
 const app = require('app')
-const ipc = require('ipc')
 const fs = require('fs')
 const path = require('path')
 const chokidar = require('chokidar')
 
-require('crash-reporter').start()
+/**
+ * Module Variable
+ * @private
+ */
+const join = path.join
 
-const mainWindow = null
+require('crash-reporter').start()
 
 const filePath = process.argv[2]
+const stylePath = process.argv[3]
+const highlightPath = process.argv[4]
+
 assert(filePath, 'no file path specified')
 
-global.baseUrl = path.relative(__dirname, path.resolve(path.dirname(filePath)));
+global.baseUrl = path.relative(__dirname, path.resolve(path.dirname(filePath)))
 if (global.baseUrl) { global.baseUrl += '/'; }
 
 const watcher = chokidar.watch(filePath, {
   usePolling: true
-})
+});
 
 // Quit when all windows are closed.
 app.on('window-all-closed', function () {
   if (process.platform !== 'darwin') app.quit()
-})
+});
 
 app.on('ready', function () {
-  window = new BrowserWindow({
+  var win = new BrowserWindow({
```

Comment:
yes totally i should have declare  `win`  in the global scope before assigning `BrowserWindow`. Anyway i prefere to not use the `window` keyword.